tutor/src/screens/my-courses/dashboard/index.tsx

```diff
@@ -0,0 +1,199 @@
+    return 0
+})  
+
+/**
+ * Component that displays the resources
+ */
+const ResourcesInfo = ({ appearanceCode, os_book_id, isFirstBlock } : {appearanceCode: string, os_book_id: string, isFirstBlock: boolean}) => {
+    const generalResources = 
+    <>
+        <Resource
+          title="Instructor Getting Started Guide"
+          info="Find information on OpenStax Tutor features and answers to common questions"
+          link={CourseInformation.gettingStartedGuide.teacher} />
+        <Resource
+          title={<span><Icon type="play-circle"/> Video Tutorials </span>}
+          info="Step by step instructions on some of the most important tasks in OpenStax Tutor"
+          link={CourseInformation.videoTutorials} />
+    </>
+    return (
+    <>
+        {isFirstBlock && generalResources}
+        {os_book_id && 
+        <Resource
+          appearanceCode={appearanceCode}
+          title="Instructor Resources"
+          info="Free resources integrated with your book. "
+          link={`https://openstax.org/details/books/${os_book_id}?Instructor%20resources`} />
+        }
+    </>
+    )
+}
+
+/**
+ * Component that displays the past courses
+ */
+const PastCourses = ({ courses }: {courses: Course[]}) => {
+    if(courses.length === 0) {
+        return <p className="no-courses-message">No past courses found.</p>
+    }
+    return map(courses, c => (<ViewCourse course={c} key={c.id} isPast={true} />))
+}
+
+/**
+ * Component that displays the current and future courses. Plus the Course Preview and the create course button
+ */
+const CurrentCourses = ({ courses, offering }: {courses: Course[], offering: Offering}) => (
+    <>
+        {map(courses, c => (<ViewCourse course={c} key={c.id}/>))}
+        <CoursePreview offering={offering} />
+        <CreateACourse />
+    </>
+)
+
+/**
+ * Component that holds the past, current and future courses. Also the resources for the course.
+ */
+const OfferingBlock = ({ offering, isFirstBlock }: {offering: Offering, isFirstBlock: boolean}) => {
+    const [tabIndex, setTabIndex] = useState(0);
+
+    const courses = useCoursesByOfferingId(offering.id)
+    const currentCourses = sortCurrentCourses(filter(courses, c => isCourseCurrent(c)))
+    const pastCourses = sortPastCourses(filter(courses, c => isCoursePast(c)))
+
+    const showTabInfo = useCallback(() => {
+        switch(tabIndex) { 
+            case 0: { 
+                return <CurrentCourses courses={currentCourses} offering={offering} />
+            } 
+            case 1: { 
+                return <PastCourses courses={pastCourses} />
+            } 
+            case 2: { 
+                return <ResourcesInfo
+                  appearanceCode={offering.appearance_code}
+                  os_book_id={offering.os_book_id}
+                  isFirstBlock={isFirstBlock} />
+            } 
+            default: { 
+               return <p>How did you get here?!</p>
```

Comment:
we should probably ask @dkhowala if she has a desired error message here, and if not use the standard PageNotFound one

Reply:
yeah :) , this was just a placeholder.